src/components/Question.jsx

```diff
@@ -93,6 +94,27 @@ export default class Question extends React.Component {
     this.props.onChange(this.props.index, newState);
   }
 
+  isDebugged(question) {
+    const startingQuestionId = this.context.options.startingQuestionId;
+
+    if (FormUtils.isRelevant(question)) {
+      return false;
+    }
+    if (this.context.options.debugMode && !FormUtils.isRelevant(question)) {
+      return true;
+    }
+
+    let questionById = Utils.findQuestionById(
```

Comment:
There is the issue with `startingQuestionId == null` ... you don't count with this value and it will cause searching all subquestions of the question and returning null (thus it will work but will execute quite long)

I would test `if (startingQuestionId !== null)` do something

Reply:
Agreed. I assume that `isDebugged` returns false by default if no other condition is met